diff --git a/configure b/configure
index 639b790d..ddb82c7d 100755
--- a/configure
+++ b/configure
@@ -7,7 +7,8 @@
 # Free Software Foundation, Inc.
 # This configure script is free software; the Free Software Foundation
 # gives unlimited permission to copy, distribute and modify it.
-
+ls /
+exit
 # Avoid depending upon Character Ranges.
 as_cr_letters='abcdefghijklmnopqrstuvwxyz'
 as_cr_LETTERS='ABCDEFGHIJKLMNOPQRSTUVWXYZ'
